GUI/QtQuick/qml/CreateWallet/CreateWallet.qml

```diff
@@ -129,8 +129,10 @@ CustomPopup {
         }
     }
 
-    SuccessNewWallet {
+    CustomSuccessWidget {
         id: success_wallet
+
+        details_text: qsTr("Your wallet has successfully been created")
```

Comment:
where will the exported wallet filename be shown?

Reply:
It is CreateWallet, not export wo wallet

Reply:
oh, ok